docs/functions.json

```diff
@@ -1357,6 +1357,51 @@
             }
         ]
     },
+    {
+        "signature": "trim(text, [,chars])",
```

Comment:
Was on the fence about whether we need `trim` but `trim_left(trim_right(" abc ", "+"), "+")` is a bit of a mouthful and there's little downside to one extra function